packages/angular/cli/src/command-builder/schematics-command-module.ts

```diff
@@ -204,14 +204,15 @@ export abstract class SchematicsCommandModule
 
                   return definition.validator(Object.values(values).map(({ value }) => value));
                 },
-                default: definition.default,
+                default: definition.multiselect ? undefined : definition.default,
```

Comment:
`default` property [is still valid](https://github.com/SBoudrias/Inquirer.js/blob/main/packages/select/README.md#options) for `select` package, used for prompt with type `list` which does not support multiselect: https://github.com/pawelfras/angular-cli/blob/62d5c330c0457a3d753359b8f520bc912ed2c3e2/packages/angular/cli/src/command-builder/schematics-command-module.ts#L197